music_assistant/providers/spotify/provider.py

```diff
@@ -267,6 +330,107 @@ async def get_playlist(self, prov_playlist_id: str) -> Playlist:
+        episode_obj = await self._get_data(f"episodes/{prov_episode_id}", market="from_token")
+        if not episode_obj:
+            raise MediaNotFoundError(f"Episode not found: {prov_episode_id}")
+        return parse_podcast_episode(episode_obj, self)
+
+    async def get_resume_position(self, item_id: str, media_type: MediaType) -> tuple[bool, int]:
+        """Get resume position for episode from Spotify."""
+        if media_type != MediaType.PODCAST_EPISODE:
+            raise NotImplementedError("Resume position only supported for podcast episodes")
+
+        if not self.sync_played_status_enabled:
+            raise NotImplementedError("Spotify resume sync disabled in settings")
+
+        try:
+            episode_obj = await self._get_data(f"episodes/{item_id}", market="from_token")
+
+            if (
+                not episode_obj
+                or "resume_point" not in episode_obj
+                or not episode_obj["resume_point"]
+            ):
+                raise NotImplementedError("No resume point data from Spotify")
+
+            resume_point = episode_obj["resume_point"]
+            fully_played = resume_point.get("fully_played", False)
+            position_ms = resume_point.get("resume_position_ms", 0)
+
+            # Apply played threshold logic
+            if not fully_played and episode_obj.get("duration_ms", 0) > 0:
+                completion_ratio = position_ms / episode_obj["duration_ms"]
+                if completion_ratio >= self.played_threshold:
+                    fully_played = True
+
+            return fully_played, position_ms
+
+        except Exception as e:
+            self.logger.debug(f"Failed to get resume position from Spotify for {item_id}: {e}")
+            raise NotImplementedError("Failed to get resume position from Spotify")
+
+    async def on_played(
+        self,
+        media_type: MediaType,
+        prov_item_id: str,
+        fully_played: bool,
+        position: int,
+        media_item: MediaItemType,
+        is_playing: bool = False,
+    ) -> None:
+        """
+        Call when an episode is played in MA.
+
+        Note: This CANNOT sync back to Spotify as there's no API for it.
+        This is just for logging/monitoring purposes.
+        """
+        if media_type != MediaType.PODCAST_EPISODE:
+            return
+
+        if not isinstance(media_item, PodcastEpisode):
+            return
+
+        # Handle case where position might be None (e.g., when marked as played in UI)
+        safe_position = position or 0
+        if media_item.duration > 0:
+            completion_percentage = (safe_position / media_item.duration) * 100
+        else:
+            completion_percentage = 0
+
+        self.logger.debug(
+            f"Episode played in MA: {prov_item_id} "
+            f"({completion_percentage:.1f}%, fully_played: {fully_played}) "
+            f"- Cannot sync back to Spotify due to API limitations"
```

Comment:
we should keep an eye on librespot because I think they are working on fixing the playback reporting

Reply:
No, but the folks from librespot are working on that - librespot will then sync the playstate to spotify (and we receive it again from the api)